### PR TITLE
Update waterfall to 18185

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

File: test/torture-s/20040409-1.c.s [hunk 1]
```diff
@@ -231,12 +236,12 @@ testu:                                  # @testu
 main:                                   # @main
 	.result 	i32
 # BB#0:                                 # %entry
-	i32.const	$push0=, 0
-                                        # fallthrough-return: $pop0
+	call    	abort@FUNCTION
```
Comment (dschuff) [inline test/torture-s/20040409-1.c.s]: @jgravelle-google this looks.... uh... yeah.

**Comment by kripken** [comment]
This PR appears to have the last commit that travis has checked (other PRs are not being checked). And the last commit did in fact alter the travis config file (to use trusty) - so perhaps that broke something?

**Comment by kripken** [comment]
@dschuff: looking in github's settings for binaryen and how travis is set up there, it appears to have your username as the owner there, so perhaps you can see what's wrong? it may be hiding info from me.

**Comment by dschuff** [comment]
I looked into this the other day. It's not limited to just Binaryen; all of the Github services for all of the wasm projects seem to be down (including Travis and IRC; but not including AppVeyor which is a webhook). So presumably the problem is GH and not Travis. Previously the travis service had JF's token and I tried switching it to mine just to see if it would do something, but it didn't. So at this point I have no idea what's going on. I even tried adding the IRC service experimentally to one of my own repos, but that didn't work either. But all the GH status pages I can find say everything is working. So... dunno, that's as far as I got.

**Pull request closed without merging**